src/common/StatsCard/StatsCard.jsx

```diff
@@ -64,6 +64,29 @@ StatsCard.Col = ({ children }) => {
 }
 StatsCard.Col.displayName = 'StatsCard.Col'
 
+StatsCard.MainCounter = ({ children, className = '', id = '', onClick = () => { } }) => {
+  return (
+    <div className={`stats__counter_main ${className}`} data-testid={id} onClick={onClick}>
```

Comment:
since a className is not required, add please it with classnames library, so we won't have something like "stats__counter_main    " (empty space after main class name)